core/geometry/src/geometry3d/Angle.ts

```diff
@@ -20,15 +20,15 @@ import { AngleProps, BeJSONFunctions, Geometry, TrigValues } from "../Geometry";
  */
 export class Angle implements BeJSONFunctions {
   /** maximal accuracy value of pi/12 (15 degrees), in radians */
-  public static readonly piOver12Radians = 0.26179938779914943653855361527329;
+  public static readonly piOver12Radians = 0.26179938779914946;
```

Comment:
@dassaf4 Are you OK with this rounding?  Essentially, this lint rule is checking that 
```js
Number(stringValue).toPrecision(originalPrecision) === stringValue;
```
So for each of these, to get the rounded values I just did something like:
```js
Number(stringValue).toString()
```
I considered just fixing each by doing:
```js
Number(stringValue).toPrecision(originalPrecision)
```
but that would have changed `piOver12Radians`, for example, to 0.261799387799149463340597776550567, which if anything seems _more_ wrong.  Besides, `(0.26179938779914946).toPrecision(33)` will get you the same value (ending in 567), so that's just how the IEEE 754 cookie crumbles I guess.

The other option would be we just disable this lint rule entirely and leave the original values, but I think a warning that you will lose some precision has _some_ merit...

Reply:
See https://github.com/iTwin/itwinjs-core/pull/5242/commits/5fa49862332df459043e582454dcda6ff8d4244b and https://github.com/iTwin/itwinjs-core/pull/5242/commits/b43e12d1fcda458eff78fa0d63b19f13d7966939 for all of the no-loss-of-precision fixes.

Reply:
I'm fine with these changes @wgoehrig. Besides, 15 digits of pi is good enough for [NASA](https://www.jpl.nasa.gov/edu/news/2016/3/16/how-many-decimals-of-pi-do-we-really-need)!